packages/docs/src/components/content/PageFeedback.tsx

```diff
@@ -19,12 +20,14 @@ export interface PageFeedbackProps {
 const PageFeedback = ({ question = 'Was this article helpful?' }: PageFeedbackProps) => {
   const [answered, setAnswered] = useState(false);
 
-  function handleClick(helpful: boolean) {
+  function handleClick(helpful: boolean, event: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
```

Comment:
What do you think about refactoring this function to pass in named arguments and updating lines 58 and 61 accordingly?

```suggestion
  function handleClick({ helpful, event }: { helpful: boolean, event: React.MouseEvent<HTMLButtonElement, MouseEvent> }) {
```

Reply:
Love it, doing it!